stake/asx/funding.py

```diff
@@ -19,6 +19,7 @@ class Action(str, Enum):
     SETTLEMENT = "SETTLEMENT"
     TRANSFER = "TRANSFER"
     WITHDRAWAL = "WITHDRAWAL"
+    ADJUSTMENT="ADJUSTMENT"
```

Comment:
please run the pre-commit hooks for formatting( black in this case )
```suggestion
    ADJUSTMENT = "ADJUSTMENT"
```